index.html

```diff
@@ -23,15 +24,109 @@
     <main>
         <section id="hero-section">
             <aside>
-            <h1>TEXT HERE</h1>
-             <button type="button">how it works</button>
+                <h1>TEXT HERE</h1>
+                <button type="button">how it works</button>
             </aside>
-            <h1>Quiick-Tik</h1> 
-            <input type="search" name="searchbox" value="search">
-            <p><small>
-            <a href="https.\\www.googlemaps.com">click here </a>to add your location
-        </small></p>
+                <h1>Quiick-Tik</h1> 
+                <input type="search" name="searchbox" value="search">
+                <p><small>
+                <a href="https.\\www.googlemaps.com">click here </a>to add your location
+                </small></p>
         </section>
+
+        
+    <article>
+        <div>
+            <h1>Featured Events</h1>
+                <ul>
+                    <li><a href="/"><h5>Featured</h5></a></li>
+                    <li><a href="/"><h5>Music</h5></a></li>
+                    <li><a href="/"><h5>Theatre</h5></a></li>
+                    <li><a href="/"><h5>Expo & Exhibitions</h5></a></li>
+                    <li><a href="/"><h5>Sport & Movies</h5></a></li>
+                </ul>
```

Comment:
Incorrect formatting.

Reply:
shouldn't he use anchor instead of ul li and h5

Reply:
beside the formatting I think he should use anchor tags only

Reply:
It's an unordered list, so I think it should be fine.